graphql2/graphqlapp/alert.go

```diff
@@ -176,6 +177,27 @@ func (a *Alert) Service(ctx context.Context, raw *alert.Alert) (*service.Service
 	return (*App)(a).FindOneService(ctx, raw.ServiceID)
 }
 
+func (m *Mutation) CreateAlert(ctx context.Context, input graphql2.CreateAlertInput) (*alert.Alert, error) {
+	a := &alert.Alert{
+		ServiceID: input.ServiceID,
+		Summary:   input.Summary,
+	}
+
+	if input.Details != nil {
+		a.Details = *input.Details
+	}
+	// An alert when created will always have triggered status
+	a.Status = alert.StatusTriggered
+
+	result, err := m.AlertStore.Create(ctx, a)
```

Comment:
I think we can just return `.Create` directly, as it already handles the `nil` return value on err.